contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -95,6 +96,9 @@ contract PrizeStrategy is PrizeStrategyStorage,
 
     prizePeriodSeconds = _prizePeriodSeconds;
     prizePeriodStartedAt = _currentTime();
+    if (_prizePeriodStart > prizePeriodStartedAt) {
```

Comment:
Let's keep it simple and just set the prize period start to the passed start *always*

Reply:
done, tests updated